DigitalExcellence/dex-backend
# Feature/139 get user from httpcontext

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist

-   [] I have added tests that prove my fix is effective or that my feature works

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. log in to the identity server
2. create a project
3. check if the project user is the current user

## Link to issue

Closes: #139
